ci/test/resources/build.yaml

```diff
@@ -52,3 +59,4 @@ steps:
     - hello_image
     - check_hello
     - hello_database
+    - inline_image
```

Comment:
Modifying the deploy this way seems a bit random.  Why not have runImage step (called something like test_inline_image?)

Reply:
The CI test checks that the deployed image is deployed. If the deploy fails, I don't think it would fail the ci test.

Reply:
I'm happy to also add something that tries to runImage, but the only observable her his whether the hello service was deployed.

Reply:
I added a runImage, but the deploy depends on the runImage now.

Reply:
The CI test should verify the deploy was successful instead of hitting the hello service.  That'll already be done by the deploy wait, and calling out to wait-for.py is kind of a bummer.  The git init thing is too clever by half.

Reply:
OK, I souped up CI to have an API endpoint for deploy status. I changed the test to wait spin until the deploy status is not None, then assert that it is success.

What's wrong with git init? I just need something that checks that `git` was actually installed.

Reply:
Git init is fine.  I mean inferring the failure of the inline image through the failure of another test.